java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Exclude PAYG (Pay-As-You-Go) instances when doing subscription matching
```

Comment:
This is not 100% true anymore as we do not fully exclude them. Maybe something like

Exclude products from PAYG ...